src/lib/utils/hooks/SynapseAPI/entity/useGetQueryResultBundle.ts

```diff
@@ -20,6 +20,10 @@ import {
 } from '../../../synapseTypes'
 import { entityQueryKeys } from './queryKeys'
 
+const sharedQueryDefaults = {
+  refetchOnWindowFocus: false,
```

Comment:
Refetch on window focus is handy for when you come back to a window after a while, but that might be a little too often for table queries.